src/backend/schemas/metrics.py

```diff
@@ -61,14 +59,20 @@ class MetricsAgent(BaseModel):
     description: str | None
 
 
+class MetricsChat(BaseModel):
+    input_nb_tokens: int
+    output_nb_tokens: int
+    search_units: int
+    model: str
+    assistant_id: str | None
```

Comment:
align on if we care about messages without assistants, and/or we should simply latch on the default assistant id

Reply:
It would still be possible to create /chat requests without an Agent and just a model deployment, so probably good to track those too

Reply:
aligned on passing in default assistant to events when the agent is not specified